src/tracebox/tracebox.cc

```diff
@@ -503,7 +505,8 @@ int doTracebox(std::shared_ptr<Packet> pkt_shrd, tracebox_cb_t *callback,
 	if (!ip)
 		return -1;
 
-	for (int ttl = 1; ttl <= hops_max; ++ttl) {
+	hops_min = hops_min == 0 ? 1 : hops_min;
```

Comment:
This won't be needed if you guard properly the initial value, i.e. make it default to one, check the value set by the user.